app/views/layout.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Chitter</title>
+  </head>
+
+  <body>
+    <h1> Chitter </h1>
+    <% if @current_user %>
+      Welcome, <%= @current_user.name %>
+    <% end %>
+     <% if flash[:notice] %>
+    <div id='notice'>
```

Comment:
Prefer semantic divs rather than <div>
https://github.com/makersacademy/chitter-challenge/blob/master/docs/review.md#use-partials-with-standard-html-conventions